src/sync/backfill-date-calc.ts

```diff
@@ -0,0 +1,40 @@
+import Logger from "bunyan";
+import { convertCommitsFromDateStringToDate } from "~/src/sync/sync-utils";
+import { SyncType } from "~/src/sync/sync.types";
+
+export const calcNewBackfillSinceDate = (
+	existingBackfillSince: Date | undefined,
+	backfillSinceInMsgPayload: string | undefined,
+	syncType: SyncType | undefined,
+	logger: Logger
+): Date | undefined  => {
+
+	if (syncType === "partial" || syncType === undefined) {
+		//do not change anything on partial sync or missing sync type on mgs body ( which means old msg before the prod deployment )
```

Comment:
why to add check for partial backfill? I believe when we'r changing the date from UI, we're backfilling commits again. Isn't it a partial backfill?

Reply:
I thought it is full? haha.

Reply:
https://github.com/atlassian/github-for-jira/blob/main/static/js/jira-configuration.js#L92